omod/pom.xml

```diff
@@ -109,9 +109,9 @@
 		<!-- End OpenMRS core -->
 		
 		<dependency>
-			<groupId>org.openmrs.directwebremoting</groupId>
+			<groupId>org.openmrs.contrib</groupId>
```

Comment:
Is there a reason why you changed the group from `directwebremoting` to `contrib`?

Reply:
@ibacher suggested I do this to maintain consistency with our other projects.